### PR TITLE
[RFC] Support for wide fields in Register and Meter

### PR DESCRIPTION
**Pull request by tatry**
This PR allows to use Registers and Meters with fields wider than 64 bits.

### CODE REVIEW

File: backends/ebpf/tests/p4testdata/wide-field-meters.p4 [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        transition accept;
+    }
+}
+
+control ingress(inout headers hdr,
+                inout empty_t user_meta,
+                in    psa_ingress_input_metadata_t  istd,
+                inout psa_ingress_output_metadata_t ostd)
+{
+    Meter<bit<128>>(16, PSA_MeterType_t.BYTES) meter1;
```
Comment (osinstom) [inline backends/ebpf/tests/p4testdata/wide-field-meters.p4:52]: Can we use sth different than IPv6 address length to verify a more corner case? The same for Registers.

**Review by osinstom — APPROVED** [review]
Minor comment, LGTM

File: backends/ebpf/tests/p4testdata/wide-field-meters.p4 (hunk 1 above)
Comment (tatry) [reply backends/ebpf/tests/p4testdata/wide-field-meters.p4:52]: It is possible I think, but I have no idea what would better fit here.

Which corner case do you mean?

Comment (osinstom) [reply backends/ebpf/tests/p4testdata/wide-field-meters.p4:52]: like bit<100>
